src/decaylanguage/dec/dec.py

```diff
@@ -1190,8 +1190,10 @@ def get_decays(parsed_file: Optional[str]) -> list[Tree]:
 
     try:
         return list(parsed_file.find_data("decay"))
-    except Exception:
-        RuntimeError("Input parsed file does not seem to have the expected structure.")
+    except Exception as err:
+        raise RuntimeError(
+            "Input parsed file does not seem to have the expected structure."
+        ) from err
```

Comment:
MyPy was complaining because these functions were supposed to return something, but they were able to complete and return None since errors must be raised - simply creating an error object does nothing, and hides the original error.

I've also made flake8-bugbear happy about explicitly indicating that this error was caused by the other one. (That's the default in Python 3, but this verifies the intent)

Reply:
Oh, good eyes! Indeed I had not seen the `as err` missing in `except Exception as err` 👍 .

Reply:
The problem is `raise` was missing. :D The `as err` and `from err` was just a minor addition to make it clear that we are not hiding the original error that bugbear wanted. `from None` would hide the original error.